src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditorFocusTracker.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * AceEditor.java
```

Comment:
Doh, thanks. (really need to just figure out an Eclipse setting that can auto-generate these)